code/API_definitions/NetworkSliceBooking.yaml

```
@@ -0,0 +1,430 @@
+openapi: 3.0.3
+info:
+  title: API for Network Slice Booking

```

Without "API", see Guidelines
```suggestion
  title: Network Slice Booking
```